src/datasource/specs/utils.spec.ts

```diff
@@ -90,21 +88,20 @@ describe('Utils', () => {
   });
 
   describe('splitTemplateQuery()', () => {
-
     // Backward compatibility
     it('should properly split query in old format', (done) => {
       let test_cases = [
         {
           query: `/alu/./tw-(nyc|que|brx|dwt|brk)-sta_(\w|\d)*-alu-[0-9{2}/`,
-          expected: ['/alu/', '/tw-(nyc|que|brx|dwt|brk)-sta_(\w|\d)*-alu-[0-9{2}/']
```

Comment:
i think something might be wrong here somehow..

i mean, i think the prettier-transform is correct.

but, the `\w\d` part looks very regex-ish, so maybe the test was incorrect already?